public/v1.html

```diff
@@ -15,13 +15,15 @@
 
 <body>
   <div class="frame">
-    <div class="splash"><a href="http://code.springroll.io"><img src="v1_assets/images/logo.png?v=1.0.3" alt="SpringRoll logo" width="128" height="128"></a><h1>SpringRoll</h1>
-      <h2
-        class="announcement green">We are excited to announce that we have just published the first major release of SpringRoll.</h2>
-      <h4 class="announcement yellow">Please point your bower.json and package.json to "^1.0.0" This release is compatible with version 0.4+ </h4>
+    <div class="splash"><a href="http://code.springroll.io"><img src="v1_assets/images/logo.png?v=1.0.3" alt="SpringRoll logo"
+             width="128" height="128"></a>
+      <h1>SpringRoll</h1>
+      <h2 class="announcement green">We are excited to announce that we have just published the first major release of
+        SpringRoll.</h2>
+      <h4 class="announcement yellow">Please point your bower.json and package.json to "^1.0.0" This release is
```

Comment:
"and package.json to "^1.0.0". 

You forgot the period at the end of the sentence :)